### ISSUE TITLE
[Accessibility] : Dual focus indicator is visible on Entire player section and its controls under 'Get the most out of Skype' heading.

### ISSUE THREAD

**Issue by Saiteja341** [videojs/video.js#7414]
**Test Environment:**
OS: Windows 10 Version (OS Build 21390.1)
Browser: Anaheim Version 92.0.902.2 (Official build) dev (64-bit)
URL : [https://www.skype.com/en/features/](url)

## Steps to reproduce
1.Open the URL in Edge Anaheim dev browser.
2.Login with valid credentials.
3.Navigate to the video player controls under 'Get the most out of Skype' heading using tab key.
4.Observe the issue.

### Expected
Dual focus indicator shouldn't be visible on Entire player section and its controls. Focus should be visible on video controls.

### Actual
Dual focus indicator is visible on Entire player section and its controls under 'Get the most out of Skype' heading.

###**User Impact**
People who depend on keyboard navigation will face difficulty if dual focus indicator is visible on the controls.

###**Guideline Reference**
https://www.w3.org/TR/WCAG21/#focus-visible

###**Attachment**
[Dual focus indicator is visible on the page.zip](https://github.com/videojs/video.js/files/7128786/Dual.focus.indicator.is.visible.on.the.page.zip)

**Comment by mister-ben** [comment]
There's just a plain video element on that page. Video.js is not being used.
